packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -505,12 +505,13 @@
         line-height: 1.45;
     }
     &--tag {
-        background-color: $color-blue;
-        color: var(--text-colored-background);
+        background-color: $color-blue-8;
+        color: $color-blue-9;
```

Comment:
how is this in dark theme?

Reply:
This part was done before and somebody over write it, I had to do it again. Based on the last confirmation that I had from the design team the same color was used for both light and dark design. I asked design team one more time to get the confirmation.